.eslintrc.js

```diff
@@ -38,7 +39,7 @@ module.exports = {
         patterns: [
           "@material-ui/*/*/*",
           "!@material-ui/core/test-utils/*",
-          "features/*/*/*",
+          // "features/*/*/*",
```

Comment:
the pattern is there on purpose to make it clear that other features should not import stuff inside other features, only what is added to the barrel file should be used

Reply:
Many thanks for reviewing! I finished.